IDisposableCodingExerciseWeek14/IDisposableCodingExerciseWeek14/Program.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IDisposableCodingExerciseWeek14
+{
+    /// <summary>
+    /// Class that attempts to complete assignment.
+    /// </summary>
+    public class Program
+    {
+        /// <summary>
+        /// Method that facilitates a comparison of not using disposal memory and disposal memory.
+        /// </summary>
+        public void Main()
+        {
+            WithoutDisposing();
+            Console.WriteLine("Done NOT using IDisposal.");
+            Console.ReadKey();
+            WithAUsingDisposal();
+            Console.WriteLine("Done using IDisposal.");
+            Console.ReadKey();
+        }
```

Comment:
Neither the Plane nor the Vehicle classes have any managed or unmanaged types that need to be disposed, so there will be no difference in the memory consumption used by them.